src/utils.ts

```diff
@@ -204,3 +205,14 @@ const urlsWebApp: any = [
 export function getCurrentPageForUrl(currentUrl: string): any {
   return urlsWebApp.find((item: any) => item.url === currentUrl);
 }
+
+export function isZohoChatOnline() {
+  const currentDate = new Date();
+  // (UTC-03:00) City of Buenos Aires 8:00 to 20:00
+  return currentDate.getUTCHours() >= 11 && currentDate.getUTCHours() < 23;
```

Comment:
It should be configurable using a settings file.

Maybe this is not the best moment to do that, in that case, what do you think about adding a TODO note?

```suggestion
  // TODO: allow to configure the schedule in a settings file.
  // (UTC-03:00) City of Buenos Aires 8:00 to 20:00
  return currentDate.getUTCHours() >= 11 && currentDate.getUTCHours() < 23;
```